test-network-function/accesscontrol/suite.go

```diff
@@ -31,11 +31,34 @@ import (
 	"github.com/test-network-function/test-network-function/pkg/tnf/handlers/serviceaccount"
 	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
 	"github.com/test-network-function/test-network-function/pkg/tnf/testcases"
+	"github.com/test-network-function/test-network-function/pkg/utils"
 	"github.com/test-network-function/test-network-function/test-network-function/common"
 	"github.com/test-network-function/test-network-function/test-network-function/identifiers"
 	"github.com/test-network-function/test-network-function/test-network-function/results"
 )
 
+const (
+	ocGetCrKindFormat = "oc get crd %s -o jsonpath='{.spec.names.kind}'"
```

Comment:
use plural or singular. i happened to notice that network-attachment-definitions.k8s.cni.cncf.io CRD has a different name as kind, which can not be used as resource type.